apps/codeforafrica/src/components/SocialMediaConnectButton/SocialMediaConnectButton.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-env browser */
+import SvgIcon from "@mui/material/SvgIcon";
+import React from "react";
+
+import FacebookIcon from "@/codeforafrica/assets/icons/Type=facebook, Size=24, Color=CurrentColor.svg";
+import GithubIcon from "@/codeforafrica/assets/icons/Type=github, Size=24, Color=CurrentColor.svg";
+import InstagramIcon from "@/codeforafrica/assets/icons/Type=instagram, Size=24, Color=CurrentColor.svg";
+import LinkedinIcon from "@/codeforafrica/assets/icons/Type=linkedin, Size=24, Color=CurrentColor.svg";
+import SlackIcon from "@/codeforafrica/assets/icons/Type=slack, Size=24, Color=CurrentColor.svg";
+import TwitterIcon from "@/codeforafrica/assets/icons/Type=twitter, Size=24, Color=CurrentColor.svg";
+import SocialMediaButton from "@/codeforafrica/components/SocialMediaButton";
+
+function getButtonIcon(name) {
+  switch (name) {
+    case "facebook":
+      return <SvgIcon sx={{ fill: "none" }} component={FacebookIcon} />;
+    case "linkedin":
+      return <SvgIcon sx={{ fill: "none" }} component={LinkedinIcon} />;
+    case "twitter":
+      return <SvgIcon sx={{ fill: "none" }} component={TwitterIcon} />;
+    case "github":
+      return <SvgIcon sx={{ fill: "none" }} component={GithubIcon} />;
+    case "instagram":
+      return <SvgIcon sx={{ fill: "none" }} component={InstagramIcon} />;
+    case "slack":
+      return <SvgIcon sx={{ fill: "none" }} component={SlackIcon} />;
+    default:
+      // TODO: if no icon found, return a generic icon ?
+      return null;
+  }
+}
+
+const SocialMediaConnectButton = React.forwardRef(
```

Comment:
```suggestion
const SocialMediaButton = React.forwardRef(
```

This would be `SocialMediaButton` then (after renaming the other one).